src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -157,7 +158,7 @@
      * Walker base classes (ReadWalker, etc.) are responsible for hooking these intervals up to
      * their particular driving data source.
      */
-    List<SimpleInterval> intervalsForTraversal;
+    List<SimpleInterval> userIntervals;
```

Comment:
To maintain as much symmetry as possible between GATKTool and GATKSparkTool, please rename the corresponding variable in GATKSparkTool to match this one. Its private (!) so the fallout should be small.

Reply:
ok